lib/capsize/deployer.rb

```diff
@@ -57,25 +60,21 @@ def invoke_task!
       end
     end
 
-    def new_execute!
+    def execute!
       config = instantiate_configuration
-      config.logger.level = options[:verbose]
       set_project_and_stage_names(config)
       find_or_create_project_dir(config.fetch(:capsize_project))
       write_deploy(config)
       write_stage(deployment.stage)
 
-      require "capsize/capsize_setup"
+      load_requirements
       capsize_setup(deployment.stage)
-      require "capistrano/deploy"
-      require 'capistrano/rvm'
-      require 'capistrano/bundler'
-      # require 'capistrano/rails/migrations'
+      set_output
       status = catch(:abort_called_by_capistrano){
-        Dir.glob('capistrano/tasks/*.rake').each { |r| import r }
         Capistrano::Application.invoke("#{deployment.stage.name}")
         Capistrano::Application.invoke(options[:actions])
       }
+      close_output
```

Comment:
This needs to be in an `ensure` block, or you have the potential of stdout to be swallowed up by an inaccessible variable.